FirebaseRemoteConfigSwift/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.0.0
+- [added] Added a new dynamic property wrapper API that enables developers to create a property wrapper property that listens to a remote config key value and automatically updates the UI component when new config are activated. (#10155)
```

Comment:
Isn't the UI component an example? Can't the property wrapper be used to trigger anything?

Alternative note: "Add property wrapper to enable dynamic monitoring of Remote Config value activations."

Reply:
I thought the main reason is to reflect UI changes which is why we need dynamic property wrapper that sort of observe if the state of a value has changed.